src/main/java/org/hl7/davinci/priorauth/ClaimEndpoint.java

```diff
@@ -599,7 +592,46 @@ private Bundle generateAndStoreClaimResponse(Bundle bundle, Claim claim, String
 
     @Override
     public void run() {
-      updateClaim(bundle, claimId, patient, disposition);
+      if (updateClaim(bundle, claimId, patient, disposition) != null) {
+        // Check for subscription
+        Map<String, Object> constraintMap = new HashMap<String, Object>();
+        constraintMap.put("claimResponseId", claimId);
+        constraintMap.put("patient", patient);
+        List<IBaseResource> subscriptions = App.getDB().readAll(Database.SUBSCRIPTION, constraintMap);
+
+        // Send notification to each subscriber
+        subscriptions.stream().forEach(resource -> {
+          Subscription subscription = (Subscription) resource;
+          SubscriptionChannelType subscriptionType = subscription.getChannel().getType();
+          if (subscriptionType == SubscriptionChannelType.RESTHOOK) {
+            // Send rest-hook notification...
+            String endpoint = subscription.getChannel().getEndpoint();
+            logger.info("SubscriptionHandler::Sending rest-hook notification to " + endpoint);
+            try {
+              OkHttpClient client = new OkHttpClient();
+              okhttp3.Response response = client.newCall(new Request.Builder().url(endpoint).build()).execute();
+              logger.fine("SubscriptionHandler::Resopnse " + response.code());
```

Comment:
Shouldn't it be SubscriptionHandler::Response?  a typo?

Reply:
Yup typo. @plarocque4 you were too slow this time